charts/posthog/README.md

```diff
@@ -16,47 +16,129 @@ This chart bootstraps a [PostHog](https://posthog.com/) deployment on a [Kuberne
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+See [Google Cloud Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/)
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+TODO
+</details>
+
+<details>
+  <summary>
+    <b>Digital Ocean</b>
+  </summary>
+<br />
+Note that there is a [one click](todo: link) option to deploy Posthog on DigitalOcean
+</details>
+
 ## Installing the chart
 
-To install the chart with the release name `posthog`, run the following:
+To install the chart with the release name `posthog` in `posthog` namespace, run the following:
 
 ```console
 helm repo add posthog https://posthog.github.io/charts-clickhouse/
 helm repo update
-helm install -f values.yaml --timeout 20m posthog posthog/posthog
+helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
+```
+
+Where the `values.yaml` has the following minimum content:
+<details>
+  <summary>
+    <b>Google Cloud</b>
+  </summary>
+<br />
+
 ```
+cloud: "gcp"
+  nginx:
+    enabled: false
+certManager:
+  enabled: false
+```
+
+</details>
+
+<details>
+  <summary>
+    <b>AWS</b>
+  </summary>
+<br />
+
+```
+cloud: "aws"
+ingress:
+  hostname: <your-hostname>
+```
+</details>
+
+<details>
+  <summary>
+    <b>Other using nginx ingress controller</b>
+  </summary>
+<br />
+
+```
+ingress:
+  hostname: <your-hostname>
+```
+
+</details>
 
 The above command deploys PostHog on the Kubernetes cluster in the default configuration.
 
-You can find an example `values.yaml` file as well as an overview of the parameters that can be configured during installation under [configuration](#configuration).
+You can find an overview of the parameters that can be configured during installation under [configuration](#configuration).
 
-> **Tip**: List all releases using `helm list`
+### Setting up DNS
 
-### Setting up a static IP on Google Cloud
+<details>
+  <summary>
+    <b> Google Cloud prerequisite </b>
+  </summary>
+<br />
+
+Set up a static IP
 
 1. Open the Google Cloud Console
 1. Go to VPC Networks > [External IP addresses](https://console.cloud.google.com/networking/addresses/list)
 1. Add new static IP with the name `posthog`
 
-### Setting up DNS
+</details>
 
 After the chart has started, you can look up the external ip via the following command:
 
 ```bash
-kubectl get svc posthog
+kubectl get svc posthog --namespace posthog
 ```
 
 Create a `CNAME` record from your desired hostname to the external IP.
 
+### Setting up email
+Outgoing email is use for password reset for example. For posthog to be able to send emails we need a login and password. Add these settings to your `values.yaml`:
+```
+email:
+  user: <your STMP login user>
+  password:  <your STMP password>
+```
+
 ## Upgrading the chart
 
 ```console
 helm repo update
-helm upgrade -f values.yaml posthog posthog/posthog
+helm upgrade -f values.yaml --timeout 20m --namespace posthog posthog posthog/posthog
```

Comment:
Why set namespace here? This only complicates instructions.

Re upgrade timeout: Why is it needed? Timeout is needed during initial install since clickhouse-operator takes a while to setup. This is not the case with upgrades.

Reply:
Can't slow migrations take longer than 5min sometimes (what about clickhouse version upgrades)?